### PR TITLE
Removing redundant team model documentation

### PR DESCRIPTION
**Pull request by markw-t**
Trimmed out old references to user-associated models and sharing via sharing link
Fixed references to the Model management window to current reality for 16.9
Removed old docs for ADO and Github task in favor of the extension/action pages which are up to date to avoid duplication and rust

### CODE REVIEW

File: docs/share-models.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ manager: jillfra
 ---
 # How to: Share team completions in Visual Studio IntelliCode
 
-IntelliCode has team completions sharing features that let you share any model you've created and automatically update users that have access when you retrain the model.
+IntelliCode has team completions sharing features that let you share repository-associated team models you've created and automatically update users that have access when you retrain the model.
 
-By creating a team completions model, you get IntelliCode completion recommendations for code that's not in the base model. Sharing a team completions model with collaborators extends the enhanced IntelliSense recommendations to everyone; they don't have to create or retrain a completions model themselves.
+By creating a team completions model, you get IntelliCode completion recommendations for code that's not in the base model. Sharing a team completions model with collaborators extends the enhanced IntelliSense recommendations to everyone; they don't have to create or retrain a team completions model themselves.
 
 ## Share a repository-associated team completions model
```
Comment (vivlimmsft) [inline docs/share-models.md:18]: the below section should be updated to reflect `Acquire team models for completion` being on by default. Maybe flip it to mention how to opt out

File: docs/custom-models.md [hunk 2]
```diff
@@ -154,72 +51,33 @@ When anyone clones and opens the codebase the model was trained on, any models a
...
-After you've trained a model, the **Share model** button appears. Click the button to copy the sharing link. From there, you can share the link with your collaborators.
-
- > [!NOTE]
- > Anyone who has the sharing link can access the model and its suggestions, so make sure that everyone who receives the link is aware of this.
-
-You can share your model with as many people as you like via the sharing feature. Team members can't retrain the model but they do see the same completion recommendations as you do.
-
-For more information about sharing completions models, see [How to: Share custom models](share-models.md).
+2.	Open the IntelliCode page by choosing **View** > **Other Windows** > **IntelliCode**
+3.	Review and accept the license terms checkbox at the bottom of the page. A machine-associated model will be trained automatically
```
Comment (vivlimmsft) [inline docs/custom-models.md:76]: this only affects the current opened solution, these steps need to be repeated for others

File: docs/custom-models.md [hunk 3]
```diff
@@ -154,72 +51,33 @@ When anyone clones and opens the codebase the model was trained on, any models a
...
 
 ## Retrain your model
```
Comment (vivlimmsft) [inline docs/custom-models.md:78]: Roll this into the above section? It only has one line now

File: docs/custom-models.md [hunk 4]
```diff
@@ -260,9 +118,9 @@ If you want to inspect the extracted data for a different codebase before trying
...
+All data you send to and receive from the IntelliCode service is transmitted over HTTPS. You must [sign in to Visual Studio](/visualstudio/ide/signing-in-to-visual-studio) in order to communicate with the service. Models can only be retrieved either by the machine which submitted the extracted data for training, or for repo-associated models by users who can prove they have access to the repository for [repository-associated models](#sharing-your-repository-associated-models). This means that your model and what is learned about your code stays private to you and your intended collaborators.
```
Comment (vivlimmsft) [inline docs/custom-models.md:123]: break this up into multiple lines, several different ideas are presented here.

File: docs/custom-models.md [hunk 5]
```diff
@@ -260,9 +118,9 @@ If you want to inspect the extracted data for a different codebase before trying
 
 ### How we secure your data
 
-Your models are private to you, those people that have the sharing links that you generate by choosing **Share completions**, and, those who have access to a [repository with an associated model](#sharing-your-repository-associated-models). 
+Your models are private to you, and, those who have access to a [repository with an associated model](#sharing-your-repository-associated-models).
```
Comment (vivlimmsft) [inline docs/custom-models.md:121]: this line is redundant with some stuff that's below

File: docs/share-models.md [hunk 6]
```diff
@@ -63,19 +37,11 @@ Suppose you have an existing codebase that contains good patterns for using type
...
+1. [Train a repo-associated model for team completions](custom-models.md#repository-associated-team-models) on the existing codebase.
 
-### Other sharing scenarios
+2. Inform your teammates to enable automatic acquisition by going to **Tools** > **Options** > **IntelliCode** > **Acquire team models for completion** in Visual Studio 2019 version 16.1 or later. When anyone clones and opens the codebase the model was trained on, any completions models associated with the configured Git remote repositories will be downloaded and activated. If you are working on a fork of the codebase, simply add the upstream codebase as a remote repository to get the completions model. Access to the repository is access to the model. When training, we collect some information about the checked-out commit. Anyone who requests that model must have the same commit in their repository and be able to produce the same information that was collected during training in order to receive the team model.
```
Comment (vivlimmsft) [inline docs/share-models.md:42]: update to reflect that the setting is on by default

File: docs/share-models.md (hunk 1 above)
Comment (markw-t) [reply docs/share-models.md:18]: Fixed, thanks Viv

File: docs/custom-models.md (hunk 3 above)
Comment (markw-t) [reply docs/custom-models.md]: Fixed, thanks Viv

File: docs/custom-models.md (hunk 4 above)
Comment (markw-t) [reply docs/custom-models.md]: Fixed, thanks Viv

File: docs/share-models.md (hunk 6 above)
Comment (markw-t) [reply docs/share-models.md]: Fixed, thanks Viv

**Comment by markw-t** [comment]
#sign-off

**Pull request merged**